src/api/java/mekanism/api/chemical/BasicChemicalTank.java

```diff
@@ -22,38 +23,88 @@ public class BasicChemicalTank implements IChemicalTank, IChemicalHandler {
     /**
      * A bi predicate that returns {@code true} for any input when the automation type is internal.
      *
      * @since 10.7.0 Previously was in ChemicalTankBuilder
+     * @deprecated Prefer using the holder variant of methods and fields.
      */
+    @Deprecated(forRemoval = true, since = "10.7.11")
     public static final BiPredicate<Chemical, @NotNull AutomationType> internalOnly = ConstantPredicates.internalOnly();
     /**
      * A bi predicate that returns {@code true} for any input when the automation type is not external.
      *
      * @since 10.7.0 Previously was in ChemicalTankBuilder
+     * @deprecated Prefer using the holder variant of methods and fields.
      */
+    @Deprecated(forRemoval = true, since = "10.7.11")
     public static final BiPredicate<Chemical, @NotNull AutomationType> notExternal = ConstantPredicates.notExternal();
     /**
      * A bi predicate that returns {@code true} for any input when the automation type is manual.
      *
      * @since 10.7.0 Previously was in ChemicalTankBuilder
+     * @deprecated Prefer using the holder variant of methods and fields.
      */
+    @Deprecated(forRemoval = true, since = "10.7.11")
     public static final BiPredicate<Chemical, @NotNull AutomationType> manualOnly = ConstantPredicates.manualOnly();
+    //TODO - 1.22: Remove the above predicates and rename the below ones to not have to have holder in the name
+
+    /**
+     * A predicate that returns {@code true} for any input.
+     *
+     * @since 10.7.11
+     */
+    public static final Predicate<Holder<Chemical>> holderAlwaysTrue = ConstantPredicates.alwaysTrue();
+    /**
+     * A predicate that returns {@code false} for any input.
+     *
+     * @since 10.7.11
+     */
+    public static final Predicate<Holder<Chemical>> holderAlwaysFalse = ConstantPredicates.alwaysFalse();
+    /**
+     * A bi predicate that returns {@code true} for any input.
+     *
+     * @since 10.7.11
+     */
+    public static final BiPredicate<Holder<Chemical>, @NotNull AutomationType> holderAlwaysTrueBi = ConstantPredicates.alwaysTrueBi();
+    /**
+     * A bi predicate that returns {@code true} for any input when the automation type is internal.
+     *
+     * @since 10.7.11
+     */
+    public static final BiPredicate<Holder<Chemical>, @NotNull AutomationType> holderInternalOnly = ConstantPredicates.internalOnly();
+    /**
+     * A bi predicate that returns {@code true} for any input when the automation type is not external.
+     *
+     * @since 10.7.11
+     */
+    public static final BiPredicate<Holder<Chemical>, @NotNull AutomationType> holderNotExternal = ConstantPredicates.notExternal();
+    /**
+     * A bi predicate that returns {@code true} for any input when the automation type is manual.
+     *
+     * @since 10.7.11
+     */
+    public static final BiPredicate<Holder<Chemical>, @NotNull AutomationType> holderManualOnly = ConstantPredicates.manualOnly();
```

Comment:
These should not be needed at all, the `ConstantPredicates` methods can be used directly

Reply:
all they're doing is a static import with extra steps